tests/test_tiatoolbox.py

```diff
@@ -408,3 +418,41 @@ def test_command_line_save_tiles(_response_ndpi, _response_svs):
     )
 
     assert save_tiles_result.exit_code == 0
+
+
+# -------------------------------------------------------------------------------------
+# Stain Normalisation
+# -------------------------------------------------------------------------------------
+
+
+def test_reinhard_normalise():
+    """Test Reinhard colour normalisation."""
+    source_img = imread("tiatoolbox/tools/stainnorm/samples/source.png")
```

Comment:
Do we need to include these images? Can we upload them on tia webpage and get access from there?

Reply:
Yes this is a good idea and was something I wanted to discuss too. Where exactly can we upload them?

Reply:
You can upload [here](https://warwick.ac.uk/fac/sci/dcs/research/tia/tiatoolbox/files)

Reply:
Addressed in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/3cfb8fe5f2b77998950484dfed6bd504af4670dd).